### ISSUE TITLE
flutter_markdown: add support for tables

### ISSUE THREAD

**Issue by AlexeySemigradsky** [flutter/flutter#8240]
Please add support for [HTML](https://developer.mozilla.org/en/docs/Web/HTML/Element/table) or [ GitHub Markdown](https://guides.github.com/features/mastering-markdown/) tables.

**Comment by ilikerobots** [comment]
@aleki I have a fork with simple markdown table support in commit  bf8fc1ce699029bf2194f335edbc9307c68800bd.  I think @abarth is completely revamping flutter_markdown in #6166, so I don't think it's useful to clean this up into a PR, but I wanted to share in case it's helpful to you.

**Comment by aleki** [comment]
@ilikerobots Thank you! I hope that `flutter_markdown` will be maintained, cuz it really important library.

**Comment by abarth** [comment]
https://github.com/flutter/flutter/pull/9044 is the PR that update flutter_markdown.  Once that lands, we can integrate the table patch.

**Comment by westy92** [comment]
https://github.com/flutter/flutter/pull/9044 has landed. 🎉

**Comment by goderbauer** [comment]
Closing as duplicate of https://github.com/flutter/flutter_markdown/issues/29, which was filed at a mire suitable place for this issue.
